src/vs/workbench/contrib/externalUriOpener/common/configuration.ts

```diff
@@ -64,6 +64,19 @@ export const externalUriOpenersConfigurationNode: IConfigurationNode = {
 	}
 };
 
+export const defaultBrowserNode: IConfigurationNode = {
+	...workbenchConfigurationNodeBase,
+	properties: {
+		'workbench.defaultBrowser': {
+			type: 'string',
+			markdownDescription: nls.localize('defaultBrowser', "Configure the default browser to use for opening http or https links"),
```

Comment:
This should explain in good detail what the available options are: either a path to the browser or one of the `open` defaults.